docs/SetupGuide_Python.md

```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
 #### Query String
 
-_TODO_
+The input binding executes the `SELECT * FROM Products WHERE Cost = @Cost` query. The Parameters argument passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. CommandType is set to `Text`, since the constructor argument of the binding is a raw query.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProducts
```

Comment:
nit: One thing I just learned about is that for "proper" markdown raw links like this should be put in <>

https://www.markdownguide.org/basic-syntax#urls-and-email-addresses

(let's try out this fancy batched suggestions too...)

```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProducts>
```

Reply:
I would include something prior to the link however such as "Sample can be found [here](link)." (especially if we plan to just include links by themselves. 

That would just be clear and concise. Also would not have to include the long links potentially.

Reply:
Yeah, I like that. I'd suggest naming the link though - e.g. 

`See the [GetProducts](...) sample`

Although that sounds a bit incomplete to me...but I can't think of anything better myself so if you come up with anything feel free to throw it out as a suggestion and we can see which one we like the best.